src/components/DocumentRenderer/ScannedDocumentRendererContainer.tsx

```diff
@@ -22,7 +22,9 @@ export const ScannedDocumentRendererContainer: FunctionComponent<NavigationProps
   const { issuers } = getData(document);
   const id = document.signature.targetHash;
   const issuedBy =
-    issuers[0]?.identityProof?.location || "Issuer's identity not found";
```

Comment:
why changing this ?

Reply:
iirc if you use opencerts-verify the identity is returned in the fragment (returned by verify function)